device/quanta/x86_64-quanta_ix1b_rglbmc-r0/plugins/psuutil.py

```diff
@@ -24,9 +25,12 @@ def show_log(txt):
 
 def exec_cmd(cmd, show):
```

Comment:
def exec_cmd(cmdargs, out_file, show):

Reply:
I updated the args and using command array instead of string.